### ISSUE TITLE
The whole Visual Studio Code is slow

### ISSUE THREAD

**Issue by schddan** [microsoft/vscode#215694]
Type: <b>Performance Issue</b>

Visual Studio Code is slow all the time, every action is taking longer than before, even the simpler ones like putting the mouse over a button.

VS Code version: Code 1.90.1 (611f9bfce64f25108829dd295f54a6894e87339d, 2024-06-11T21:02:43.666Z)
OS version: Linux x64 6.5.0-35-generic snap
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 3700U with Radeon Vega Mobile Gfx (8 x 2671)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|4, 2, 1|
|Memory (System)|5.65GB (2.28GB free)|
|Process Argv|--force-user-env --no-sandbox --use-gl=angle --use-angle=swiftshader --crash-reporter-id f383280a-780d-4dd0-bb07-fbb0fb2002d9|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|wayland|
</details><details>
<summary>Process Info</summary>

```
CPU %	Mem MB	   PID	Process
    0	   162	 13220	code main
    0	    41	 13222	   zygote
    0	    41	 13223	   zygote
    0	   376	 13302	window [1] (Bem-vindo - Frontend - Visual Studio Code)
    1	    93	 13932	window [2] (Relator de Problemas)
    0	    64	 13257	   utility-network-service
    0	   151	 13537	shared-process
    0	     0	 13951	     /bin/sh -c /usr/bin/ps -ax -o pid=,ppid=,pcpu=,pmem=,command=
    0	     0	 13952	       /usr/bin/ps -ax -o pid=,ppid=,pcpu=,pmem=,command=
    0	   145	 13853	extensionHost [1]
    0	    87	 13863	fileWatcher [1]
```

</details>
<details>
<summary>Workspace Info</summary>

```
|  Window (Bem-vindo - Frontend - Visual Studio Code)
|    Folder (Frontend): 100 files
|      File types: jsx(26) ttf(19) png(9) css(9) svg(8) otf(8) json(2) cjs(1)
|                  gitignore(1) md(1)
|      Conf files: package.json(1);
```

</details>
<details><summary>Extensions (14)</summary>

Extension|Author (truncated)|Version
---|---|---
c-cpp-compile-run|dan|1.0.58
vscode-language-pack-pt-BR|MS-|1.89.2024050109
debugpy|ms-|2024.6.0
python|ms-|2024.8.1
vscode-pylance|ms-|2024.6.1
jupyter|ms-|2024.5.0
remote-wsl|ms-|0.88.2
cmake-tools|ms-|1.18.42
cpptools|ms-|1.20.5
cpptools-extension-pack|ms-|1.3.0
makefile-tools|ms-|0.9.10
material-icon-theme|PKi|5.4.2
LiveServer|rit|5.7.9
cmake|twx|0.0.17

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
jchc7451:31067544
dsvsc021:30996838
724cj586:31013169
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
pythonprt:31056678
dwnewjupytercf:31046870

```

</details>

**Comment by shelvesdragon** [comment]
This is likely caused by the temporary fix in version 1.90.1 on the snap package that switched the OpenGL backend to software emulation in order to prevent the crashes reported with version 1.90.0 in #212494 . If it is related, your options for now are mentioned in this comment: https://github.com/microsoft/vscode/issues/212494#issuecomment-2168072242
Hope this helps!

**Comment by deepak1556** [comment]
Merging to https://github.com/microsoft/vscode/issues/212494, please follow the issue for updates.
